pkg/cluster/types.go

```diff
@@ -42,6 +42,7 @@ type Upgrader interface {
 	UpdateWorkerAgentPool(ctx context.Context, cs *api.OpenShiftManagedCluster, app *api.AgentPoolProfile, suffix string) *api.PluginError
 	EtcdRestoreDeleteMasterScaleSet(ctx context.Context, cs *api.OpenShiftManagedCluster) *api.PluginError
 	EtcdRestoreDeleteMasterScaleSetHashes(ctx context.Context, cs *api.OpenShiftManagedCluster) *api.PluginError
+	ResetUpdateBlob(cs *api.OpenShiftManagedCluster) error
```

Comment:
should go in GenevaActions interface - need to rebase

Reply:
The action `ForceUpdate` is already in the `GenevaActions` interface. That method uses `simpleUpgrader.ResetUpdateBlob`

Reply:
sorry, my comment was garbage